compiler/src/dotty/tools/dotc/typer/VarianceChecker.scala

```diff
@@ -132,14 +132,16 @@ class VarianceChecker()(implicit ctx: Context) {
           ctx.debuglog(s"Skipping variance check of ${sym.showDcl}")
         case tree: TypeDef =>
           checkVariance(sym, tree.pos)
```

Comment:
On `tests/pos/Iterable.scala` we get an error inside this call (going through `tp.classParents`), because `sym` contains the parens but without annotations!